package.json

```diff
@@ -9,13 +9,16 @@
   "dependencies": {
     "apollo-client": "^1.9.0",
     "auth0-js": "^8.11.2",
+    "dotenv": "^4.0.0",
```

Comment:
These three added dependencies are only required for development so they should live in `devDependencies` in this file.